notebook/nbconvert/handlers.py

```diff
@@ -116,13 +122,25 @@ def get(self, format, path):
             filename = os.path.splitext(name)[0] + resources['output_extension']
             self.set_header('Content-Disposition',
                                'attachment; filename="%s"' % escape.url_escape(filename))
-
         # MIME type
         if exporter.output_mimetype:
             self.set_header('Content-Type',
                             '%s; charset=utf-8' % exporter.output_mimetype)
+        self.finish(output) 
+
+    @web.authenticated
+    def get(self, format, path):
+        self.call_nbconvert(format, path, config=self.config)
+    
+    @web.authenticated
+    def post(self, format, path):
+        c = Config(self.config)
+        json_upload = self.get_json_body()
+        json_config = json.loads(json_upload.get("config",{}))
+        c.merge(json_config)
+        nb_content = json.dumps(json_upload["notebook"])
+        self.call_nbconvert(format, path, config=c, content=nb_content, post=True)
```

Comment:
I think that's for symmetry with `get` that was defined before. The "only" difference between `get` and `post` is that it gets an extra config object. There is also an efficiency question that if the notebook is on the server already we don't **need** to send it on the wire (and that's how it works currently). In the end we _want_ to be able to post the notebook content and this is part of the transition effort.